packages/playwright-ct-vue2/registerSource.mjs

```diff
@@ -156,4 +158,11 @@ window.playwrightUnmount = async element => {
   element.remove();
 };
 
-const instanceKey = Symbol('instanceKey');
+window.playwrightSetProps = async (element, props) => {
+  const component = /** @type {any} */(element)[instanceKey];
+  if (!component)
+    throw new Error('Component was not mounted');
+
+  for (const [key, value] of Object.entries(props))
+    component.$children[0][key] = value;
```

Comment:
This does not look like something documented to update the UI upon the changes. Is this something VTU uses?